scripts/third_party_size_check.py

```diff
@@ -33,18 +33,27 @@ def _get_skip_files_list():
 
     Returns:
         list. The list of files which are to be skipped.
-    """
-    with open('./app.yaml', 'r') as app_yaml:
-        try:
-            app_yaml_dict = yaml.safe_load(app_yaml)
-        except yaml.YAMLError as yaml_exception:
-            print yaml_exception
-        skip_files_list = app_yaml_dict.get('skip_files')
-
-        skip_files_list = [os.getcwd() + '/' + skip_files_dir
-                           for skip_files_dir in skip_files_list]
 
-    return skip_files_list
+    Raises:
+        yaml.YAMLError if failed to parse app_dev.yaml.
+        IOError if failed to open app_dev.yaml in read mode.
```

Comment:
When would this happen? The try except for that looks too large in scope (We try to have as few statements within the try except block as possible). I think it is ok as we are catching one specific exception type.

Reply:
The IO error? It's happening at the moment:
```
Traceback (most recent call last):
  File "scripts/third_party_size_check.py", line 102, in <module>
    _check_third_party_size()
  File "scripts/third_party_size_check.py", line 77, in _check_third_party_size
    skip_files_list = _get_skip_files_list()
  File "scripts/third_party_size_check.py", line 37, in _get_skip_files_list
    with open('./app.yaml', 'r') as app_yaml:
IOError: [Errno 2] No such file or directory: './app.yaml'
```

Reply:
I think this happens due to the change to use app_dev.yaml.

Reply:
Yes, therefore, we should future proof this, right? Because this was not caught earlier in the original PR.